src/components/BookPage/AddReviewForm/index.js

```diff
@@ -21,63 +26,47 @@ function Form(props) {
 
   function handleSubmit(event) {
     event.preventDefault();
-    setNameError("");
-    setRatingError("");
-    setReviewError("");
-
-    let isValid = true;
-
-    if (name.trim() === "") {
-      setNameError("Name field is required");
-      isValid = false;
-    }
-
-    if (rating === "") {
-      setRatingError("Rating field is required");
-      isValid = false;
-    }
 
-    if (review.trim() === "") {
-      setReviewError("Review field is required");
-      isValid = false;
-    }
-
-    if (isValid) {
-      fetch("https://book-swap-api.dev.io-academy.uk/api/reviews", {
-        method: "POST",
-        body: JSON.stringify({
-          name: `${name}`,
-          rating: rating,
-          review: `${review}`,
-          book_id: id,
-        }),
-
-        mode: "cors",
-        headers: {
-          "Content-Type": "application/json",
-          Accept: "application/json",
-        },
-      })
-        .then((res) => res.json())
-        .then((data) => {
-          console.log(data);
+    fetch("https://book-swap-api.dev.io-academy.uk/api/reviews", {
+      method: "POST",
+      body: JSON.stringify({
+        name: name,
+        rating: rating,
+        review: review,
+        book_id: props.id,
+      }),
+      mode: "cors",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+      },
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        if (data.errors) {
+          setNameError(data.errors.name);
+          setRatingError(data.errors.rating);
+          setReviewError(data.errors.review);
+        } else {
+          setNameError("");
+          setRatingError("");
+          setReviewError("");
           setName("");
           setRating("");
           setReview("");
           setReviewCharCount(0);
```

Comment:
This is getting a bit chunky now, you should move all these into a function 'clearForm' or something